### PR TITLE
feat(view): #MC-168 add response toasters

### PR DESCRIPTION
**Pull request by alicedraillard**
## Describe your changes
toasters indicate if processes were successful

## Checklist tests
try to update and create external calendars and check the toaster responses

## Issue ticket number and link
[MC-168](https://jira.support-ent.fr/browse/MC-168)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/resources/public/ts/directives/calendar-item/calendar-item.ts [hunk 1]
```diff
@@ -59,25 +59,42 @@ class Controller implements ng.IController, IViewModel {
...
-        this.$timeout(() : IPromise<void> => {
-            this.calendarService.checkExternalCalendarSync(this.$scope.vm.calendar)
-                .then((r:AxiosResponse) => {
-                    if(r.data['isUpdating'] == false) {
+        try {
+            await this.calendarService.updateExternalCalendar(this.$scope.vm.calendar);
+            this.$timeout(() : IPromise<void> => {
+                this.calendarService.checkExternalCalendarSync(this.$scope.vm.calendar)
+                    .then((r:AxiosResponse) => {
+                        if(r.data['isUpdating'] == false) {
```
Comment (JinDouang) [inline src/main/resources/public/ts/directives/calendar-item/calendar-item.ts:67]: Obligé de faire ['isUpdating']? ton data est généralement en mode "any" donc tu peux faire directement r.data.isUpdating

Une autre alternative serait de typer ton data :  
```suggestion
                        if((<ton_type>res.data).isUpdating == false) {
```

**Review by JinDouang — CHANGES_REQUESTED** [review]

File: src/main/resources/public/ts/directives/calendar-item/calendar-item.ts [hunk 2]
```diff
@@ -59,25 +59,42 @@ class Controller implements ng.IController, IViewModel {
         if($event) $event.stopPropagation();
         this.loading = true;
         safeApply(this.$scope);
-        this.calendarService.updateExternalCalendar(this.$scope.vm.calendar);
-        this.$timeout(() : IPromise<void> => {
-            this.calendarService.checkExternalCalendarSync(this.$scope.vm.calendar)
-                .then((r:AxiosResponse) => {
-                    if(r.data['isUpdating'] == false) {
+        try {
+            await this.calendarService.updateExternalCalendar(this.$scope.vm.calendar);
```
Comment (alicedraillard) [inline src/main/resources/public/ts/directives/calendar-item/calendar-item.ts:63]: indentation --> l84

File: src/main/resources/public/ts/directives/calendar-item/calendar-item.ts (hunk 1 above)
Comment (alicedraillard) [reply src/main/resources/public/ts/directives/calendar-item/calendar-item.ts:67]: le truc c'est que data a le type any

Comment (qpic-cgi) [reply src/main/resources/public/ts/directives/calendar-item/calendar-item.ts:67]: Rien ne t'empêche de le typer / caster !

**Review by qpic-cgi — APPROVED** [review]

**Review by qperie — APPROVED** [review]

**Review by sjollois — APPROVED** [review]

**Review by JinDouang — APPROVED** [review]

**Pull request merged**